rfd/0037-desktop-access-protocol.md

```diff
@@ -200,11 +200,13 @@ This message contains a mouse wheel update. Sent from client to server.
 #### 9 - error
 
 ```
-| message type (9) | message_length uint32 | message []byte
+| message type (9) | message_length uint32 | message []byte | fatal bool
```

Comment:
Any compatibility issues here? We need to support the old message too since w_d_s may be older than the proxy.

Reply:
(Ah yes, I knew there was something I was forgetting to add to the description). 

I was going to write that it should work fine, because the first section of the message structure remains the same and in practice we only ever decode Errors on the frontend, whose client would just ignore the additional `fatal`. But now that I'm double checking, I realized that we're calling `decodeClipboardData` now from the proxy (via `ReadRaw()`), which will error if the message is in the old format.

Will experiment and come up with a way to account for that.